AOJ/03/03_C.py

```diff
@@ -0,0 +1,21 @@
+a = []
+while True:    
+    x,y = map(int,input().split())
+    if x == 0 and y == 0:
+        break
+    else:
+        a += [ (x,y) ]
+
+y = 0
+while True:    
+    if y == len(a):
+        break
+    else:
+        if a[y][0] < a[y][1]:
+            print(str(a[y][0]) + ' ' + str(a[y][1]))
+            y += 1    
+        else:
+            print(str(a[y][1]) + ' ' + str(a[y][0]))
+            y += 1
```

Comment:
if文とelse文で、`y += 1` の処理が重複しているので、条件分の外に出すようにしてください。
```suggestion    
        else:
            print(str(a[y][1]) + ' ' + str(a[y][0]))
        y += 1
```